libs/remix-astwalker/tests/newTests.ts

```diff
@@ -130,12 +130,12 @@ tape("New ASTWalker", (t: tape.Test) => {
 	 However, for non-typescript uses, we add this test which casts
 	 to an AST to check that there is a run-time check in walkFull.
       */
-      astWalker.walkFull(<AstNode>legacyNode, (node: AstNode) => {
+      astWalker.walkFull(<any>legacyNode, (node: AstNode) => {
         count += 1;
       });
     }
-    t.throws(badCall, /first argument should be an ast/,
-      "passing legacyAST fails");
+    // t.throws(badCall, /first argument should be an ast/,
```

Comment:
any good reason to remove these tests?

Reply:
Because when legacyAST is passed we don't show an error to keep it compatible to old compilers